### DISCUSSION TITLE
macos + позиционирование окна far2l при старте

### DISCUSSION THREAD

**Discussion by DEF29** [elfmz/far2l#1364] [Q&A] [answered]
Можно ли управлять окном far2l при старте?

Оно у меня съезжает настолько в нижний правый угол, что только его фрагмент и виден - как я только с ним не боролся и пока решения нет - временно использую Moom чтобы центрировать окно far2l но для этого приходится после запуска нажимать специальную комбинацию клавиш для центрирования...

**Comment by DEF29** [comment]
Вот так выглядит рабочий стол когда я запускаю far2l

![image](https://user-images.githubusercontent.com/37115984/192918074-fbc2201d-bc63-4cec-ae48-3d398573e751.png)

**Comment by elfmz** [comment, marked as answer]
теперь состояние окна полностью должны сохраняться при сохранении общих настроек (Shift+F9)

**Reply by DEF29** [reply]
Огромнейшая вам благодарность! Ежедневную радость приносят плоды ваших трудов, наш любимый FAR все лучше с каждым днем!

**Comment by DEF29** [comment]
Поправка!
У меня Mojave и я ставил far2l-2.5.0-beta-MacOS-10.11.dmg
У меня пока не выявлено сбоев нигде
Друг скачал и поставил far2l-2.5.0-beta-MacOS-10.15.dmg
и у него появилась проблема с SFTP:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/37115984/213760125-583f9dae-dc82-4523-8318-879162ab0fb3.png">
У друга Big Sur

**Reply by elfmz** [reply]
А это в новой libssh задепрекейтили ssh-rsa и ssh-dss как несекьюрные. Наверное надо сделать настройку, но пока не вышла версия с настройкой, можно воркэраунднуть таким образом
- в настроках подключения ткнуть Protocol options и далее включить галочку  Use OpenSSH configuration files
- открыть в текстовом редакторе /etc/ssh/**ssh_config** (не путать с sshd_config!) и добавить в конец строчку: 
`HostkeyAlgorithms ssh-ed25519,ssh-ed25519-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-rsa-cert-v01@openssh.com,ssh-dss-cert-v01@openssh.com,ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ssh-rsa,ssh-dss`

**Reply by DEF29** [reply]
Благодарю за рецепт! Сработало)
